Фича "Отзывы" и "Лайки для отзывов"

disochkin opened the pull request:
минимальная функциональность только проходит тесты Postman

Shihovcev committed:
добавлена функциональность «Общие фильмы»

# Conflicts:
#	pom.xml
#	src/main/java/ru/yandex/practicum/filmorate/controller/FilmController.java
#	src/main/java/ru/yandex/practicum/filmorate/interfaces/FilmRepository.java
#	src/main/java/ru/yandex/practicum/filmorate/repository/JdbcFilmRepository.java
#	src/main/java/ru/yandex/practicum/filmorate/service/FilmService.java

Shihovcev committed:
добавлена функциональность «Общие фильмы»

Shihovcev committed:
добавлена функциональность «Общие фильмы»

disochkin committed:
MVP
проходит тесты Postman

disochkin committed:
MVP
проходит тесты Postman

disochkin committed:
fix code style

Allion591 commented on pom.xml:
```diff
@@ -68,6 +68,11 @@
+			<artifactId>h2</artifactId>
+			<scope>runtime</scope>
+		</dependency>
 	</dependencies>
```
А этой зависимости не было вроде, подскажи для чего она нужна?

disochkin replied:
в application.properties пришлось добавить
spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
и без этой зависимости не работало

disochkin committed:
добавил spring.jpa.database-platform=org.hibernate.dialect.H2Dialect в тестовый application_test.properties

disochkin committed:
добавил сортировку по id для пользователей

disochkin committed:
изменил Collection на List

disochkin committed:
изменил Collection на List

disochkin committed:
добавил тестовые данные

Allion591 committed:
Добавил контроллеры для обработки поисковых запросов

Allion591 committed:
Добавил методы пля поиска фильмов в интерфейс

Allion591 committed:
Добавил методы пля поиска фильмов в класс для работы с БД

Allion591 committed:
Добавил методы пля поиска фильмов в сервисный слой

disochkin committed:
вернул Collections для Users
убрал org.hibernate.dialect.H2Dialect из application.properties и application_test.properties
убрал зависимости spring-boot-starter-test, com.h2database из pom.xml

disochkin committed:
вернул Collections для Users
убрал org.hibernate.dialect.H2Dialect из application.properties и application_test.properties
убрал зависимости spring-boot-starter-test, com.h2database из pom.xml

Shihovcev committed:
Merge branch 'develop' into add-reviews

Shihovcev committed:
добавлена функциональность «Общие фильмы»

Shihovcev committed:
добавлена функциональность «Общие фильмы»

Shihovcev committed:
добавлена функциональность «Общие фильмы»

disochkin committed:
изменения после исправления кофликтов

disochkin committed:
add users

disochkin committed:
add users

disochkin committed:
add users

disochkin committed:
add users

disochkin committed:
add users

disochkin committed:
add users

disochkin committed:
add users

disochkin committed:
add users

Vatakat92 reviewed (approved):
все норм
